README.md

```diff
@@ -1 +1,16 @@
-# sal-backend
\ No newline at end of file
+# sal-backend
+
+Mac OS 
+in sal/sal-backend
+mkdir src
+cd src
+ln -s /Users/preeti.dave/sal/sal-backend/aws/ aws
+GOOS=LINUX go build -o github.com/build/user github.com/sal-group/sal-backend/user/user.go
+zip build/user and upload to lambda functions code
+
+To Test: 
+curl --request GET https://sf7z9xf5i9.execute-api.us-east-2.amazonaws.com/dev/users
```

Comment:
lets not include these publicly for now ... as these repos are accessible publicly